src/Covenant.CycloneDx/Covenant.CycloneDx.csproj

```diff
@@ -11,7 +11,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="CycloneDX.Core" Version="5.4.0" />
+    <PackageReference Include="CycloneDX.Core" Version="7.1.0" />
```

Comment:
This PR seems to update a bunch of components. Consider adding a description to this PR clarifying whether that's a case of "Updating these while we're in here" or whether these are necessary to achieve the end goal.